lasso/roadway.py

```diff
@@ -1940,6 +1942,10 @@ def write_roadway_as_fixedwidth(
         link_ff_df, link_max_width_dict = self.dataframe_to_fixed_with(
             self.links_metcouncil_df[link_output_variables]
         )
+
+        if drive_only:
+            link_ff_df.loc[link_ff_df['drive_access'] == 1]
```

Comment:
I think this is just doing a selection; not actually assigning it to a variable.

Reply:
How do I filter on the variable and return the dataframe?

Reply:
The way you filter is fine, you just need to assign it to a object, in this case, either overwrite or create new.  like this:

`link_ff_df = link_ff_df[link_ff_df['drive_access'] == 1]`